template.yaml

```diff
@@ -511,6 +510,21 @@ Resources:
             Path: /funding-sources/{identifier}
             Method: get
 
+  NvaCristinFetchBiobankFunction:
+    Description: Function for fetching a Biobank by identifier
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: cristin-biobank
+      Handler: no.unit.nva.biobank.fetch.FetchBiobankHandler::handleRequest
+      Role: !GetAtt ReadCristinAuthenticationRole.Arn
```

Comment:
Remove this. We do not need authenticated biobank queries if it is not required in cristin biobank fetch